tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest==3.8.2
+Click==7.0
+envparse==0.2.0
+PyGithub==1.43.2
```

Comment:
Also to avoid repetitive requirements, you can add `-e .` here and I'd put this as `test-requirements.txt` in the top-level dir.